models/articles.models.js

```diff
@@ -1,5 +1,29 @@
 const db = require("../db/connection");
 
+exports.fetchArticles = async () => {
+  try {
+    const query = `SELECT 
+        articles.author,
+        articles.title,
+        articles.article_id,
+        articles.topic,
+        articles.created_at,
+        articles.votes,
+        articles.article_img_url,
+        CAST(COUNT(comments.body) AS INT) AS comment_count
+      FROM articles
+      LEFT JOIN comments
+      ON articles.article_id = comments.article_id
+      GROUP BY articles.article_id
+      ORDER BY articles.created_at DESC;
+    `
+    const results = await db.query(query)
+    return results.rows
+  } catch (error) {
+    throw error
+  }
```

Comment:
You don't need this catch in your model, the one in your controller will catch any errors from invoking this model there